### DISCUSSION TITLE
User como nullable nos controllers

### DISCUSSION THREAD

**Discussion by antonrafael** [balta-io/7196#38] [Perguntas e Respostas] [answered]
Dúvida sobre a API no dotnet 6
Testando a Api criada através do postman, todas as requisições estavam retornando o seguinte erro:

**"The Field User is Required"**

Para não precisar passar um usuário no Body da Request, alterei a propriedade User para nullable em todos os Commands, desta forma:

```
public class CreateTodoCommand : Notifiable, ICommand

    {    
        public string Title { get; set; }
        public DateTime Date { get; set; }
        public string? User { get; set; }
     }
```

public **string?** User { get; set; }   

isso resolveu o problema, gostaria de saber se há algum problema de segurança ao fazer isso, ou se essa é realmente a solução correta?

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @rafaelThorus, pode ser assim como fez ou assim:
```csharp
public string User { get; set; } = string.Empty;
```

**Reply by antonrafael** [reply]
ook, obrigado pelo auxílio
